Sources/CoreXLSX/Worksheet/CellQueries.swift

```diff
@@ -62,6 +62,14 @@ public extension Cell {
     return sharedStrings.items[index].text
   }
 
+  /// Returns the value of the cell as a RichText,
+  /// given `sharedStrings` argument.
+  func richStringValue(_ sharedStrings: SharedStrings) -> RichText? {
+    guard type == .sharedString, let index = value.flatMap(Int.init) else { return nil }
+
+    return sharedStrings.items[index].richText.first
```

Comment:
```suggestion
  func richStringValue(_ sharedStrings: SharedStrings) -> [RichText] {
    guard type == .sharedString, let index = value.flatMap(Int.init) else { return [] }

    return sharedStrings.items[index].richText
```

Since `richText` is an array, I think some of our users would want to get the whole array, the first argument can always be read from it than needed.